pkg/cli/admin/upgrade/status/health.go

```diff
@@ -194,19 +201,43 @@ type displayItem struct {
 	message     string
 	description string
 	reference   string
+
+	resourceKindPad int
+	// resourceKinds contains keys of resources map
+	resourceKinds []string
```

Comment:
Should we include the group too?  Sometimes `kind` is not sufficient to be unique, e.g. in a `TechPreviewNoUpgrade` cluster:

```console
$ oc api-resources | grep 'NAME\|^machines '
NAME                                  SHORTNAMES                                                                             APIVERSION                                      NAMESPACED   KIND
machines                              ma                                                                                     cluster.x-k8s.io/v1beta1                        true         Machine
machines                                                                                                                     machine.openshift.io/v1beta1                    true         Machine
```

And folks need to use `oc get machines.machine.openshift.io` if they want the machine-API type, because `oc get machines` will give them the cluster-API type.

Reply:
/hold

I'll check if I can address that easily

Reply:
I agree, adding groups is a good idea.

Reply:
done in [733af23](https://github.com/openshift/oc/pull/1737/commits/733af23bebbc52b5978d76a5a84419ff0291f699)

Reply:
Groups added:

/hold cancel